src/odemis/gui/comp/overlay/world.py

```diff
@@ -1465,6 +1464,10 @@ def stop_updating_tool(self):
             if self._ask_user_for_text or not self._edit_label_end:
                 self._edit_text()
                 self._ask_user_for_text = False
+            # In case no line is drawn, the user can still type some text
+            if self.v_end_pos is None:
+                offset = self.cnvs.get_half_buffer_size()
+                self.p_start_pos = self.p_end_pos = self.cnvs.view_to_phys(self.v_start_pos, offset)
```

Comment:
I don't understand why this is needed. The `GenericGadgetLine.start_dragging()` method (now) takes care of initialising the v_start/v_end and they are immediately used to compute p_start/p_end. So this `if` should never be triggered. Or am I missing a corner case?